File: sdkwork-app-sdk-kotlin/src/main/kotlin/com/sdkwork/app/TextSuggestVO.kt
package com.sdkwork.app

data class TextSuggestVO(
    val createdAt: String? = null,
    val updatedAt: String? = null,
    val originalText: String? = null,
    val suggestedText: String? = null,
    val replacementCount: Int? = null,
    val positions: List<ReplacementPosition>? = null,
    val alternativeSuggestions: List<String>? = null
)

File: sdkwork-app-sdk-typescript/src/types/text-suggest-vo.ts
import type { ReplacementPosition } from './replacement-position';

/** 文本替换建议响应 */
export interface TextSuggestVO {
  /** 创建时间 */
  createdAt?: string;
  /** 更新时间 */
  updatedAt?: string;
  /** 原始文本 */
  originalText?: string;
  /** 建议文本 */
  suggestedText?: string;
  /** 替换次数 */
  replacementCount?: number;
  /** 替换位置 */
  positions?: ReplacementPosition[];
  /** 其他建议 */
  alternativeSuggestions?: string[];
}
